newIDE/app/src/UI/MarkdownText.js

```diff
@@ -57,15 +59,19 @@ export const MarkdownText = (props: Props) => {
   );
 
   return (
-    <ReactMarkdown
-      escapeHtml
-      source={props.source}
-      className={classNames({
-        'gd-markdown': true,
-        [gdevelopTheme.markdownRootClassName]: true,
-        'standalone-text-container': props.isStandaloneText,
-      })}
-      renderers={markdownCustomRenderers}
-    />
+    <I18n>
+      {({ i18n }) => (
+        <ReactMarkdown
+          escapeHtml
+          source={i18n._(props.source)}
```

Comment:
Not sure if it's safe to use i18n._ here, as we can get some already translated strings passed. (unfortunately, the MessageDescriptor is just a `string | Object`, so does not offer strong typing guarantees.

Reply:
As discussed, I've added a new prop `translatableSource` to be used alongside `source`.

I couldn't get the smart typing to work though.
When I define a component as `A | B` as soon as I start testing if prop `props.source` exists, it considers it's a component of type `A` and complains about using `B.translatableSource`. 
I searched for a better way to type, but nothing really obvious.

Reply:
Ok, no big deal, these union types are always weird to work with.